piecedirectory/piecedirectory_test.go

```diff
@@ -29,17 +26,17 @@ func TestPieceDirectory(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
 	defer cancel()
 
-	t.Run("leveldb", func(t *testing.T) {
+	t.Run("piecedirectory with leveldb", func(t *testing.T) {
 		bdsvc, err := svc.NewLevelDB("")
 		require.NoError(t, err)
 		testPieceDirectory(ctx, t, bdsvc)
 	})
 
-	t.Run("yugabyte", func(t *testing.T) {
-		svc.SetupYugabyte(t)
-		bdsvc := svc.NewYugabyte(svc.TestYugabyteSettings)
-		testPieceDirectory(ctx, t, bdsvc)
-	})
+	//t.Run("yugabyte", func(t *testing.T) {
```

Comment:
I realize we also want more tests on Yugabyte, but not 100% sure this is the right place for that.